_config.yml

```diff
@@ -49,9 +49,9 @@ other_nodegit_versions:
   - 0.2.7
 
 # Build settings
-markdown: redcarpet
+markdown: kramdown
```

Comment:
Is there any specific reason why we needed to change the markdown or the highlighter settings?

Reply:
I was unable to get the website to generate without doing `bundle update github-pages` because of an error with the json dependency. Bundle updating fixed that error by updating everything in the Gemfile.lock file. The update got rid of `redcarpet` and `pygments`, so `script/server wouldn't run because of missing dependencies. I googled what the alternatives were, and noticed rouge and kramdown were already in the updated Gemfile.lock. So I changed it in the config file, and `script/server` ran successfully, letting me generate the website.